### ISSUE TITLE
Please make an app

### ISSUE THREAD

**Issue by HunterFloper** [clickerultimate/clickerultimate.github.io#12]
Is there an app for this

**Comment by clickerultimate** [comment]
No, this is not planned. It's technically possible to port browser games like these to other platforms like Steam, but I have not considering looking into it at the moment.
